### ISSUE TITLE
Calendar example fails to build in docs

### ISSUE THREAD

**Issue by mattpap** [bokeh/bokeh#12262]
`make html` fails with:
```
bokeh-plot:: error:

Traceback (most recent call last):
  File "/home/mateusz/repo/bokeh/bokeh/application/handlers/code_runner.py", line 231, in run
    exec(self._code, module.__dict__)
  File "/home/mateusz/repo/bokeh/examples/models/file/calendars.py", line 96, in <module>
    months = [ [ make_calendar(2014, 3*i + j + 1) for j in range(3) ] for i in range(4) ]
  File "/home/mateusz/repo/bokeh/examples/models/file/calendars.py", line 96, in <listcomp>
    months = [ [ make_calendar(2014, 3*i + j + 1) for j in range(3) ] for i in range(4) ]
  File "/home/mateusz/repo/bokeh/examples/models/file/calendars.py", line 96, in <listcomp>
    months = [ [ make_calendar(2014, 3*i + j + 1) for j in range(3) ] for i in range(4) ]
  File "/home/mateusz/repo/bokeh/examples/models/file/calendars.py", line 28, in make_calendar
    firstweekday = list(day_abbrs).index(firstweekday)
ValueError: 'Mon' is not in list
```
but I can run this example manually and it works fine. All dependencies are up-to-date.

**Comment by bryevdv** [comment]
Well, `day_abbrs` is just an import from `calendar` so I don't see how this can be anything other than an upstream change or problem. That said, this example:

* is long / complicated / lots of code
* demonstrates atypical / uncommon usage (i.e. not plotting)
* uses a package that is deprecated: https://pypi.org/project/Calendar/

So my actual preference here would just be to remove this example altogether

**Comment by mattpap** [comment]
I was hoping that PR #12265 would fix this, but no. Maybe other dependencies are responsible, but I don't have time right now to investigate this.

**Comment by mattpap** [comment]
> Well, day_abbrs is just an import from calendar so I don't see how this can be anything other than an upstream change or problem.

I'm running sphinx and this standalone example in the same environment. It works when run standalone and doesn't in sphinx, there must be more to this. Though this is the only one that fails (I needed to remove it and re-run build, as sphinx reports one error at a time).

**Comment by mattpap** [comment]
> demonstrates atypical / uncommon usage (i.e. not plotting)

That's true for many other examples, e.g. elements. However, having such examples is important, because they show not just what this system was designed for, but also what it can actually do beyond its primary function. That being said, I can move it to bokehjs' examples (under integration tests).

**Comment by mattpap** [comment]
This was fixed in PR #13753.
